### ISSUE TITLE
add ability to add hooks to common webelement actions, e.g. to "slow" them for "demo reasons"

### ISSUE THREAD

**Issue by extsoft** [yashaka/selene#7]
It will be great to have "demo mode" for test. Just make a pause after action to allow see the system behavior - useful thing when make a demo for customer etc.

**Comment by yashaka** [comment]
Here is an example of how this feature is implemented in Selenide (at least its 2.7 version):

```java
        WebDriverRunner.addListener(
                new AbstractWebDriverEventListener() {

                    public void beforeClickOn(WebElement element, WebDriver driver) {

                        debugLog("CLICKING: " + element);
                        sleep(100);
                    }


                    public void beforeFindBy(By by, WebElement element, WebDriver driver) {
                        sleep(100);
                    }

                });
```

**Comment by yashaka** [comment]
There should be also an ability to provide one hook for all actions.
Use cases:
- slow down everything "for demo reasons"
- add waitForAngular waiting before all actions
